test/test_standard_pipelines.py

```diff
@@ -0,0 +1,326 @@
+    join_node = JoinDocuments(join_mode="concatenate")
+    p = Pipeline()
+    p.add_node(component=es, name="R1", inputs=["Query"])
+    p.add_node(component=dpr, name="R2", inputs=["Query"])
+    p.add_node(component=join_node, name="Join", inputs=["R1", "R2"])
+    results = p.run(query=query)
+    assert len(results["documents"]) == 3
+
+    # test join_node with reader
+    join_node = JoinDocuments()
+    p = Pipeline()
+    p.add_node(component=es, name="R1", inputs=["Query"])
+    p.add_node(component=dpr, name="R2", inputs=["Query"])
+    p.add_node(component=join_node, name="Join", inputs=["R1", "R2"])
+    p.add_node(component=reader, name="Reader", inputs=["Join"])
+    results = p.run(query=query)
+    #check whether correct answer is within top 2 predictions
+    assert results["answers"][0].answer == "Berlin" or results["answers"][1].answer == "Berlin"
+
+
+def test_query_keyword_statement_classifier():
+    class KeywordOutput(RootNode):
+        outgoing_edges = 2
+
+        def run(self, **kwargs):
+            kwargs["output"] = "keyword"
+            return kwargs, "output_1"
+
+    class QuestionOutput(RootNode):
+        outgoing_edges = 2
+
+        def run(self, **kwargs):
+            kwargs["output"] = "question"
+            return kwargs, "output_2"
+
+    pipeline = Pipeline()
+    pipeline.add_node(
+        name="SkQueryKeywordQuestionClassifier",
+        component=SklearnQueryClassifier(),
+        inputs=["Query"],
+    )
+    pipeline.add_node(
+        name="KeywordNode",
+        component=KeywordOutput(),
+        inputs=["SkQueryKeywordQuestionClassifier.output_2"],
+    )
+    pipeline.add_node(
+        name="QuestionNode",
+        component=QuestionOutput(),
+        inputs=["SkQueryKeywordQuestionClassifier.output_1"],
+    )
+    output = pipeline.run(query="morse code")
+    assert output["output"] == "keyword"
+
+    output = pipeline.run(query="How old is John?")
+    assert output["output"] == "question"
+
+    pipeline = Pipeline()
+    pipeline.add_node(
+        name="TfQueryKeywordQuestionClassifier",
+        component=TransformersQueryClassifier(),
+        inputs=["Query"],
+    )
+    pipeline.add_node(
+        name="KeywordNode",
+        component=KeywordOutput(),
+        inputs=["TfQueryKeywordQuestionClassifier.output_2"],
+    )
+    pipeline.add_node(
+        name="QuestionNode",
+        component=QuestionOutput(),
+        inputs=["TfQueryKeywordQuestionClassifier.output_1"],
+    )
+    output = pipeline.run(query="morse code")
+    assert output["output"] == "keyword"
+
+    output = pipeline.run(query="How old is John?")
+    assert output["output"] == "question"
+
+
+@pytest.mark.parametrize(
+        "retriever,document_store",
+        [
+            ("embedding", "faiss"),
+            ("embedding", "milvus"),
+            ("embedding", "elasticsearch"),
+        ],
+        indirect=True,
+)
+def test_document_search_pipeline(retriever, document_store):
```

Comment:
we have two tests with the same name here (also had that before but let's fix that now)